crates/bevy_sprite/src/mesh2d/color_material.rs

```diff
@@ -28,7 +27,7 @@ impl Plugin for ColorMaterialPlugin {
         app.world.resource_mut::<Assets<ColorMaterial>>().insert(
             Handle::<ColorMaterial>::default(),
             ColorMaterial {
-                color: LegacyColor::rgb(1.0, 0.0, 1.0),
+                color: Color::srgb(1.0, 0.0, 1.0),
```

Comment:
```suggestion
                color: Color::FUCHSIA,
```
`pub const FUCHSIA: Srgba = Srgba::new(1.0, 0.0, 1.0, 1.0);`